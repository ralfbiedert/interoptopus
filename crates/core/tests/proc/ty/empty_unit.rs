use interoptopus::ffi_type;

#[ffi_type]
struct Foo;

fn main() {}
