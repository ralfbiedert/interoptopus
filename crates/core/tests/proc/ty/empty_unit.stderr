error: Empty structs are not allowed unless marked as #[ffi(opaque)] or #[ffi(service)].
 --> tests/proc/ty/empty_unit.rs:4:8
  |
4 | struct Foo;
  |        ^^^
